da/lazyledger/encoding.go

```diff
@@ -0,0 +1,20 @@
+package lazyledger
+
+import (
+	authtypes "github.com/cosmos/cosmos-sdk/x/auth/types"
+	"github.com/lazyledger/lazyledger-app/app/params"
+)
+
+// RegisterAccountInterface registers the authtypes.AccountI interface to the
+// interface registery in the provided encoding config
+func RegisterAccountInterface(conf params.EncodingConfig) params.EncodingConfig {
+	conf.InterfaceRegistry.RegisterInterface(
+		"cosmos.auth.v1beta1.BaseAccount",
+		(*authtypes.AccountI)(nil),
+	)
+	conf.InterfaceRegistry.RegisterImplementations(
+		(*authtypes.AccountI)(nil),
+		&authtypes.BaseAccount{},
+	)
+	return conf
+}
```

Comment:
do we need this function here, as opposed to using the [app's](https://github.com/celestiaorg/lazyledger-app/blob/0e60861ff7f5548d52391f5aa57014e2554bbbff/app/params/encoding.go#L21)?

Reply:
Good catch